docs/docs/integrations/document_loaders/web_base.ipynb

```diff
@@ -66,7 +66,10 @@
    "source": [
     "from langchain_community.document_loaders import WebBaseLoader\n",
     "\n",
-    "loader = WebBaseLoader(\"https://www.espn.com/\")"
+    "loader = WebBaseLoader(\"https://www.espn.com/\")\n",
```

Comment:
i dont think this should be documented here, it should just be prevalent in the reference docs